site/docs/guide/getting-started.md

```diff
@@ -150,6 +150,90 @@ in your terminal before you execute `node bot.js`.
 This makes it easier to debug your bot.
 :::
 
+## Getting Started on Bun
+
+> This guide assumes that you have [Bun](https://bun.sh) installed.
+> If you don't know what these things are, check out our [Introduction](./introduction)!
+
+Create a new project and install the `grammy` package.
+Do this by opening a terminal and typing:
+
+```sh
+# Create a new directory and change into it.
+mkdir my-bot
+cd my-bot
+
+# Run bun init to scaffold a new project.
+bun init --yes
```

Comment:
Does Bun really require a setup script?

Reply:
Not really, Bun just uses it for creating default files like README, package.json, tsconfig and index.ts.

I don't know about older versions, but the new ones add the package without init

Reply:
tsconfig can't be created by bun, unless it installs and runs `tsc`? Because the rest of the website depends on having the TypeScript default config and I don't think I want to trust bun on making sure that these two files are always in sync, so we have to tell people to run `tsc` themselves.

Reply:
> tsconfig can't be created by bun, unless it installs and runs `tsc`? Because the rest of the website depends on having the TypeScript default config and I don't think I want to trust bun on making sure that these two files are always in sync, so we have to tell people to run `tsc` themselves.

I don't quite understand the question. Bun does not use tsc for compilation. It runs Typescript immediately

Reply:
This is incorrect. It should be this:
```json
{
  "compilerOptions": {
    "target": "es2016",
    "module": "commonjs",
    "esModuleInterop": true,
    "forceConsistentCasingInFileNames": true,
    "strict": true,
    "skipLibCheck": true
  }
}
```

Reply:
Why?

Reply:
Because the rest of the docs require it. Every single of the ~700 code examples are tested on both Deno and Node and they work with the config created in the getting started guide. So we either make sure that Bun is compatible with the Node examples, or else we will have to re-create all code examples with a Bun tab. Even if it turns out to the mostly the same in most cases, we have to least test everything.

I don't care about bun and I don't want to spend time on this. I am not going install it. I also want to be able to edit the docs myself, and write new docs. This is not possible if I have to test everything on bun first. I won't accept any contributions that prevent me from changing the docs in the future, or that force me to ask for help for every single edit.

The bottom line is that the bun setup MUST work with the existing Node code. It must type-check with the same TS config, and it must run correctly.

Reply:
> The bottom line is that the bun setup MUST work with the existing Node code. It must type-check with the same TS config, and it must run correctly.

I understand your indignation about this runtime, but I have not had any problems with it before when working with it.

Reply:
Actually, nobody will test if it runs correctly, people are just going to watch it fail when they use Bun. But at least we can make sure that it type-checks.

Reply:
> This is incorrect. It should be this:
> 
> ```json
> {
>   "compilerOptions": {
>     "target": "es2016",
>     "module": "commonjs",
>     "esModuleInterop": true,
>     "forceConsistentCasingInFileNames": true,
>     "strict": true,
>     "skipLibCheck": true
>   }
> }
> ```

I'll check all examples with Bun if you want so
![image](https://github.com/user-attachments/assets/38b025a0-4b76-40c8-8f9d-14069a0a8a47)

Reply:
> > The bottom line is that the bun setup MUST work with the existing Node code. It must type-check with the same TS config, and it must run correctly.
> 
> I understand your indignation about this runtime, but I have not had any problems with it before when working with it.

Sure but you also did not test everything